digits/views.py

```diff
@@ -90,6 +90,11 @@ def home(tab=2):
                     'url': flask.url_for(
                         'digits.pretrained_model.views.new'),
                     },
+                'access-model-store': {
+                    'title': 'Retrieve from Model Store',
+                    'id': 'retrieveModelStore',
+                    'url': '/static/store.html',
```

Comment:
Let's make this an actual Flask route and not just a link to a static HTML file.